quinn-udp/src/windows.rs

```diff
@@ -178,6 +178,15 @@ impl UdpSocketState {
                 encoder.push(WinSock::IPPROTO_IPV6, WinSock::IPV6_ECN, ecn);
             }
 
+            // Segment size is a u32 https://learn.microsoft.com/en-us/windows/win32/api/ws2tcpip/nf-ws2tcpip-wsasetudpsendmessagesize
```

Comment:
this should be documented close to `CMSG_LEN` L374 and also taken into account in the assert in the `UdpSocketState::new()` function